health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -54,6 +55,132 @@ export const UICustomizations = {
+          });
+          return res;
+
+        }
+        const fun = async ()=>{
+          res = await callSearch();
+        }
+        // fun();
+        switch (key) {
+          case "HIERARCHY_NAME":
+            return row?.hierarchyType;
+            break;
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)